### ISSUE TITLE
Missing Proptypes rule does not go inside if inside the render

### ISSUE THREAD

**Issue by mikaelcastellani** [yannickcr/eslint-plugin-react#144]
Hi,

I noticed that the props check rule does not go inside a if inside a render method. Here is my use case :
```javascript
render() {
  if (...) {
    return <B x={this.props.x} y={this.props.y} />;
  } else {
    return <A />;
  }
}
```

With this code I have no 'Missing PropTypes' warning for x and y, but when I remove the if and just return, I have the warning (as expected).

Thanks
